blocks/network-item/network-item.js

```diff
@@ -94,78 +99,103 @@ function getText(block) {
   return div;
 }
 
-export default function decorate(block) {
-  const blockCfg = readBlockConfig(block);
-  const image = getImage(block);
-  const text = getText(block);
-  const hasImage = !!blockCfg.image;
-
-  let { title } = blockCfg;
+function createTitle(title) {
+  const titleH3 = document.createElement('h3');
+  titleH3.classList.add('title');
+  titleH3.innerText = title;
+  return titleH3;
+}
 
-  if (Array.isArray(blockCfg.title)) {
-    title = title.join('\n');
+function extractVideoFromContent(content, videoImage) {
+  const video = content.querySelector('p.button-container > a').parentNode;
+  const div = document.createElement('div');
+  div.classList.add('network-item-img');
+  div.append(video.children[0]);
+
+  if (videoImage) {
+    const picture = videoImage.querySelector('picture');
+    picture.classList.add('network-item-img', 'video-image');
+    const a = div.querySelector('a');
+    a.replaceChildren(picture);
+    a.classList.remove('button');
   }
 
-  const tags = [...blockCfg.tags.split(',')];
+  content.removeChild(video);
+  return div;
+}
 
+function getVideoImage(block) {
+  const div = getNamedValueFromTable(block, 'Video-image');
+  if (!div) return null;
+  div.classList.add('network-item-img');
+  return div;
+}
+
+export default function decorate(block) {
+  const blockCfg = readBlockConfig(block);
+  const hasMultimediaContent = !!blockCfg.image || block.classList.contains('video');
```

Comment:
Just wondering if the 'explicit' cast to boolean (with `!!`) is necessary? Wouldn't the fact that you're using the or operator `||` already implicitly do that for you?

Reply:
From what I gathered, no, it will not cast it to a boolean implicitly
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR

`"Cat" || false; // t || f returns "Cat"`

The checks later on only depend on the truthy value of `hasMultimediaContent` so I guess I could have it not be a boolean sometimes and it would still work, but I think I like it more like this, but I can change it if need be